lib/workers/repository/process/lookup/types.ts

```diff
@@ -49,6 +49,7 @@ export interface LookupUpdateConfig
   replacementNameTemplate?: string;
   replacementVersion?: string;
   extractVersion?: string;
+  vulnerabilityFix?: string;
```

Comment:
Shouldn't this be defined in the `PackageRule` interface instead (where `isVulnerabilityAlert` is too)? Same below

Reply:
Unfortunately adding it there doesn't remove the need to have it in the two types in this file too